packages/dashboard/src/API.tsx

```diff
@@ -18,7 +18,6 @@ export interface PluginServices extends DashboardPluginServices {
 }
 export interface PluginMessages extends DashboardPluginMessages {
     Wallet: typeof WalletMessages
-    Transak: typeof import('../../mask/dist/src/plugins/Transak/messages').PluginTransakMessages
```

Comment:
Please do not access `TransakMessages`, but use ways like how we handle `Wallet`.